tools/DevDiagnostics/DevHome.DevDiagnostics/Views/ClipboardMonitoringWindow.xaml.cs

```diff
@@ -1,16 +1,22 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-using DevHome.DevDiagnostics.Helpers;
-using WinUIEx;
+using DevHome.Common.Extensions;
+using DevHome.DevDiagnostics.Controls;
+using Microsoft.UI.Xaml;
 
 namespace DevHome.DevDiagnostics.Views;
 
-public sealed partial class ClipboardMonitoringWindow : WindowEx
+public sealed partial class ClipboardMonitoringWindow : ThemeAwareWindow
 {
     public ClipboardMonitoringWindow()
     {
         InitializeComponent();
-        Title = CommonHelper.GetLocalizedString("ClipboardMonitorWindowTitle");
+    }
+
+    private void ThemeAwareWindow_Closed(object sender, WindowEventArgs args)
+    {
+        var barWindow = Application.Current.GetService<PrimaryWindow>().DBarWindow;
+        barWindow?.RemoveRelatedWindow(this);
     }
```

Comment:
Why can't this be handled in the base class?

Reply:
Because BarWindow is also derived from the same base class, and it probably isn't right for the base class to have knowledge of its derived classes.